docs/_description_templates/template-statement.md

```diff
@@ -0,0 +1,23 @@
+# Statement name (for example, SHOW USER)
+
+What does the query initiated by the statement do.
+
+## Syntax
```

Comment:
I'm not sure if one code block deserves its own header. As far as I remember currently just "Syntax:" is more popular.